x-pack/plugins/security_solution/public/common/containers/alerts/use_alert_prevalence_from_process_tree.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useEffect, useState } from 'react';
+
+import { useQuery } from 'react-query';
+import { useHttp } from '../../lib/kibana';
+
+// import { DEFAULT_MAX_TABLE_QUERY_SIZE } from '../../../../common/constants';
+
+// import { useGlobalTime } from '../use_global_time';
+// import { TimelineId } from '../../../../common/types';
+// import { useDeepEqualSelector } from '../../hooks/use_selector';
+// import { inputsSelectors } from '../../store';
+
+export const DETECTIONS_ALERTS_COUNT_ID = 'detections-alerts-count';
+
+interface UseAlertPrevalenceOptions {
+  parentEntityId: string | string[] | undefined | null;
+  timelineId: string;
+  signalIndexName: string | null;
+}
+
+interface UserAlertPrevalenceFromProcessTreeResult {
+  loading: boolean;
+  alertIds: undefined | string[];
+  count?: number;
+  error: boolean;
+}
+
+interface ProcessTreeAlertPrevalenceResponse {
+  alertIds: string[];
+}
+
+export function useAlertPrevalenceFromProcessTreeActual(
+  processEntityId: string
+): UserAlertPrevalenceFromProcessTreeResult {
+  const http = useHttp();
+  const query = useQuery<ProcessTreeAlertPrevalenceResponse>(
+    ['getAlertPrevalenceFromProcessTree', processEntityId],
+    () => {
+      return http.get<ProcessTreeAlertPrevalenceResponse>('/TBD', {
```

Comment:
@kqualters-elastic This would be the module and the place to integrate the endpoint into 🎉